plugins/together/modelgauge/annotators/llama_guard_annotator.py

```diff
@@ -264,18 +264,33 @@ def translate_response(
         self, request: TogetherCompletionsRequest, response: TogetherCompletionsResponse
     ) -> LlamaGuardAnnotation:
         assert len(response.choices) == 1
-        lines = response.choices[0].text.split()
-        # This is all very strict, might want to be less so.
-        if lines[0] == "safe":
-            return LlamaGuardAnnotation(is_safe=True, violation_categories=[])
-        elif lines[0] == "unsafe":
-            categories = []
-            assert len(lines) > 1
-            for raw in lines[1].split(","):
-                categories.append(self.decoder[raw])
-            return LlamaGuardAnnotation(is_safe=False, violation_categories=categories)
-        else:
-            raise AssertionError("Unexpected response: ", response.choices[0].text)
+        response_str = response.choices[0].text
+        return translate_llama_guard_response(
+            response=response_str, version=self.llama_guard_data, decoder=self.decoder
+        )
+
+
+def translate_llama_guard_response(
```

Comment:
(3) Refactor to expose the llama guard response translation to make it a reusable method. We wanted to reuse this logic in our test